source/MRMesh/MRConfig.h

```diff
@@ -27,14 +27,15 @@ class Config
 
     MRMESH_API const std::string& getAppName() const;
 
+    // writes current config to file
+    MRMESH_API void writeToFile();
```

Comment:
mb add comment that is imlicitly called from destructor

Reply:
added comment